### ISSUE TITLE
Question about running time of cryoDRGN2 and NeRF

### ISSUE THREAD

**Issue by zhouzhenghong-gt** [zhonge/cryodrgn#99]
Thanks for your great work!! And I have two question about cryoDRGN2.
1. Will training schedule of cryoDRGN2 lead to a long training time?
2. In your cryoDRGN2 paper, you mention that gradient descent can't optimize pose, have you tried some NeRF related methods such as nerf--, BARF or GNeRF?
Thanks for your time!

**Comment by zhonge** [comment]
Thanks for asking.

1) yes it will lead to a much longer training time than without pose
search. Despite that, cryoDRGN2 is significantly faster and more accurate
than cryoDRGN1's pose inference (described in the ICLR paper), though we're
thinking of ways to make it even faster

2) yes, first order gradient based optimization has been technically
available in the main cryoDRGN software since the initial release
(--do-pose-sgd) but requires different training settings and doesn't lead
to a large difference in many cases, I believe due to the high amount of
noise in cryo-EM images. It could be evaluated further though.
